controllers/v1/sprint.go

```diff
@@ -241,3 +268,27 @@ func (ctrl SprintController) GetSprintMemberSummary(c *gin.Context) {
 
 	c.JSON(status, response)
 }
+
+// GetTrails is method to get the all trails related to a particular sprint
+func (ctrl SprintController) GetTrails(c *gin.Context) {
+	sprintID := c.Param("sprintID")
+	retroID := c.Param("retroID")
+	userID, _ := c.Get("userID")
+	sprintIDInt, errConversion := strconv.Atoi(sprintID)
+
+	if errConversion != nil {
+		c.AbortWithStatusJSON(http.StatusForbidden, gin.H{})
+		return
+	}
+	if !ctrl.PermissionService.UserCanAccessSprint(retroID, sprintID, userID.(uint)) {
+		c.AbortWithStatusJSON(http.StatusForbidden, gin.H{})
+		return
+	}
+
+	trails, status, err := ctrl.TrailService.GetTrails(uint(sprintIDInt))
+
+	if err != nil {
```

Comment:
Why is this error handled differently, i.e., why nothing is returned in the response?

Reply:
Done